packages/mainmenu-extension/src/index.ts

```diff
@@ -535,19 +535,23 @@ function createKernelMenu(
   commands.addCommand(CommandIDs.shutdownAllKernels, {
     label: trans.__('Shut Down All Kernels…'),
     isEnabled: () => {
-      return !app.serviceManager.sessions.running().next().done;
+      return !app.serviceManager.kernels.running().next().done;
```

Comment:
Is this change needed?

```suggestion
      return !app.serviceManager.sessions.running().next().done;
```

Reply:
Not necessarily. I changed them for consistency with what is done in the sidebar. Should I keep it as it was?